src/sagemaker/estimator.py

```diff
@@ -3066,6 +3064,34 @@ def _validate_and_set_debugger_configs(self):
                     )
                     self.debugger_hook_config = False
 
+    def _validate_mwms_config(self, distribution):
```

Comment:
Estimator.py file is becoming too lengthy. Considering the importance of this module and the way it's growing is there room to move this validation logic outside of this module?